options/db_options.cc

```diff
@@ -241,11 +241,6 @@ static std::unordered_map<std::string, OptionTypeInfo>
                    skip_checking_sst_file_sizes_on_db_open),
           OptionType::kBoolean, OptionVerificationType::kNormal,
           OptionTypeFlags::kNone}},
-        {"new_table_reader_for_compaction_inputs",
-         {offsetof(struct ImmutableDBOptions,
-                   new_table_reader_for_compaction_inputs),
-          OptionType::kBoolean, OptionVerificationType::kNormal,
-          OptionTypeFlags::kNone}},
```

Comment:
Need to keep this, change verification type to `OptionVerificationType::kDeprecated`, and replace `offsetof()` call with zero.